test/test-rotatesTetrominos.js

```diff
@@ -1,32 +1,194 @@
 var assert = require('chai').assert;
 var tetris = require('../tetris');
-describe("rotate L-tetromino", function (){
+describe("Class tetramino", function (){
     class Tetromino {
-        constructor(tetramino, phase, coordinates){
-            this.typeTetramoni = tetramino
-            this.initialRotationPhase = phase;
-            this.initialCoordinates = coordinates;
+        constructor(typeTetramoni, initialRotationPhase, initialCoordinates){
+            this.typeTetramoni = typeTetramoni
+            this.initialRotationPhase = initialRotationPhase;
+            this.initialCoordinates = initialCoordinates;
         }
-        _rotateTetromino(){
-					
+        rotateTetromino(typeOfTetromino, rotationPhaseOfTetromino, tetrominoCoordinates) {
+            let shift = this.determineShift(typeOfTetromino, rotationPhaseOfTetromino)
```

Comment:
you already have this functionality in tetris.rotateTetromino
Your algorithm is different